### PR TITLE
Storybook: Add BlockAlignmentMatrixControl Stories and update README

### PR DESCRIPTION
**Pull request by Sukhendu2002**
Part of https://github.com/WordPress/gutenberg/issues/67165

## What?
This PR will add stories for BlockAlignmentMatrixControl component in the Storybook.

## Testing Instructions
1. Run npm run storybook:dev
2. Open the storybook on http://localhost:50240/
3. Check the BlockAlignmentMatrixControl stories.

## Screenshots or screencast
image

### CODE REVIEW

**Comment by Sukhendu2002** [comment]
Hi @t-hamano, consider this PR instead of this https://github.com/WordPress/gutenberg/pull/67292

I have applied all the suggested changes, Feel free to take a look whenever you have time. Thanks!

File: packages/block-editor/src/components/block-alignment-matrix-control/index.js [hunk 1]
```diff
@@ -11,6 +11,35 @@ import {
...
+ *   );
+ * }
+ * ```
+ *
+ * @param {Object}   props            Component props.
+ * @param {string}   props.label      Label for the control. Defaults to 'Change matrix alignment'.
+ * @param {Function} props.onChange   Function to execute upon change of matrix state.
+ * @param {string}   props.value      Content alignment location. One of: 'center', 'center center', 'center left', 'center right', 'top center', 'top left', 'top right', 'bottom center', 'bottom left', 'bottom right'.
+ * @param {boolean}  props.isDisabled Whether the control should be disabled.
+ * @return {Element}                    The BlockAlignmentMatrixControl component.
```
Comment (t-hamano) [inline packages/block-editor/src/components/block-alignment-matrix-control/index.js]:
```suggestion
 * @param {Object}   props            Component props.
 * @param {string}   props.label      Label for the control. Defaults to 'Change matrix alignment'.
 * @param {Function} props.onChange   Function to execute upon change of matrix state.
 * @param {string}   props.value      Content alignment location. One of: 'center', 'center center',
 *                                    'center left', 'center right', 'top center', 'top left',
 *                                    'top right', 'bottom center', 'bottom left', 'bottom right'.
 * @param {boolean}  props.isDisabled Whether the control should be disabled.
 * @return {Element}                  The BlockAlignmentMatrixControl component.
```

It is a good idea to use consistent indentation and line breaks for long comments.

File: packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+					'Renders a control for selecting block alignment using a matrix of alignment options.',
+			},
+		},
+	},
+	argTypes: {
+		label: {
+			control: 'text',
+			table: {
+				type: { summary: 'string' },
+				defaultValue: { summary: 'Change matrix alignment' },
```
Comment (t-hamano) [inline packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js]:
```suggestion
				defaultValue: { summary: "'Change matrix alignment'" },
```

File: packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+				type: { summary: 'boolean' },
+				defaultValue: { summary: 'false' },
+			},
+			description: 'Whether the control should be disabled.',
+		},
+		value: {
+			control: { type: null },
+			table: {
+				type: { summary: 'string' },
+				defaultValue: { summary: 'center' },
```
Comment (t-hamano) [inline packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js]:
```suggestion
				defaultValue: { summary: "'center'" },
```

**Review by t-hamano — COMMENTED** [review]
Thanks for the update! I left a small suggestion at the end and it's working fine 👍

File: packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js (hunk 2 above)
Comment (Sukhendu2002) [reply packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js]: While committing the code, the double quote (") automatically changes to single quote (')

File: packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js (hunk 3 above)
Comment (Sukhendu2002) [reply packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js]: same for this one

File: packages/block-editor/src/components/block-alignment-matrix-control/index.js (hunk 1 above)
Comment (Sukhendu2002) [reply packages/block-editor/src/components/block-alignment-matrix-control/index.js]: Hi @t-hamano, thanks for the review, for the return can I follow this one https://github.com/WordPress/gutenberg/issues/22891#issuecomment-2543030487

File: packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js (hunk 2 above)
Comment (t-hamano) [reply packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js]: That's strange. I can commit normally in my environment:

https://github.com/user-attachments/assets/ffcfcb5d-d72e-4c26-b58c-9e23aedf7616

In your environment, what operation changes it to single quotes?

Comment (Sukhendu2002) [reply packages/block-editor/src/components/block-alignment-matrix-control/stories/index.story.js]: Sorry for the trouble. It was my editor's fault, not a pre-commit hook as I thought. I’ve updated the PR.

**Review by t-hamano — APPROVED** [review]
LGTM 👍

**Pull request merged**